src/hearings/hearings.routes.ts

```diff
@@ -254,7 +255,8 @@ export const ROUTES: Routes = [
         component: HearingCreateEditSummaryComponent,
         canActivate: [HealthCheckGuard],
         data: {
-          title: 'HMCTS Hearings | Request Hearing | Check Answers'
+          title: 'HMCTS Hearings | Request Hearing | Check Answers',
+          isChildRequired: true
```

Comment:
we only need to set the isChildRequired: true for the otherPanelRoles, but for other LoV it should be still the default false/no.